BlargBucket/ViewModels/ReadMeViewModel.swift

```diff
@@ -0,0 +1,35 @@
+//
+//  ReadMeViewModel.swift
+//  BlargBucket
+//
+//  Created by Nick Smillie on 12/23/14.
+//  Copyright (c) 2014 Luke Deniston. All rights reserved.
+//
+
+import Foundation
+
+class ReadMeViewModel: NSObject {
+	
+	dynamic var readMeMarkDown: String?
+	var readMeAttributed: NSAttributedString?
+	let repo = AppDelegate.sharedInstance().activeRepo
```

Comment:
There's a lot to be said for avoiding singletons when possible. It would require you to change the initializer for this class to something like `init(repo:Repository)`, which mean you'll have to change where you initialize it inside of `ReadMeViewController`. The main advantage we would get out of rewriting it like that is it would let us write tests more easily.